docs/changelog.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../CHANGES.rst
```

Comment:
We probably introduce towncrier for this.

Reply:
Interesting, I'd been looking for a tool to do something along these lines.

If you're going to use `towncrier` to generate a `CHANGES.rst` file, then this will continue to work fine. Did you have something different in mind?

Reply:
Yeah, that what I was think of.